Delta debug orfs integration test

oharboe opened the pull request.

oharboe committed:
test: deltaDebug.py integration test belongs in ORFS

this avoids OpenROAD depending on ORFS to run integration
tests.

Once this PR is merged, delete tools/OpenROAD/etc/test_delta_debug.sh

Signed-off-by: Øyvind Harboe <oyvind.harboe@zylin.com>

maliberty commented on flow/test/test_delta_debug.sh:
```diff
@@ -0,0 +1,21 @@
+echo "Testing $latest_file"
+. ../env.sh
+rm -rf test_delta_debug
+mkdir test_delta_debug
```
Write into test/results as noted in your gitignore PR

oharboe replied:
Fixed.

oharboe committed:
test: deltaDebug now uses results/delta_debug/ folder

this is covered by .gitignore

The output is on the order of e.g. results/asap7/mock-array/ as it is a compressed
version of that data with "make issue"

$ du -hs results/delta-debug/
110M	results/delta-debug/

Signed-off-by: Øyvind Harboe <oyvind.harboe@zylin.com>
